internal/services/aws_reservation_service.go

```diff
@@ -151,7 +151,8 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Return response payload
-	if err := render.Render(w, r, payloads.NewAWSReservationResponse(reservation)); err != nil {
+	unused := make([]*models.ReservationInstance, 0, 0)
```

Comment:
why unused?

Reply:
are you good here Adi? I agree it's not a perfect code, but I do not see better solution either, so I'd just merge as is.